lib/vector-core/src/mapping/query/arithmetic.rs

```diff
@@ -75,7 +75,7 @@ impl Function for Arithmetic {
     fn execute(&self, ctx: &Event) -> Result<QueryValue> {
         let left = match self.left.execute(ctx)? {
             QueryValue::Value(value) => value,
-            query => {
+            query @ QueryValue::Regex(_) => {
```

Comment:
Another one where I think `if let` would be clearer. Also below in all of these similar constructs where we only care about one enum variant.

Reply:
Reversed in d949fa1